Update dependencies

Justinzobel opened the pull request:
Currently targeting 6.9 as there is no 6.10 branch. If someone can please create that, thank you.

Commit:
Update dependencies

Justinzobel commented:
Pending QtWebEngine PR merging: https://github.com/flathub/io.qt.qtwebengine.BaseApp/pull/393

shyamnathp reviewed (approved):
Hey, thanks for the patch. When i created the 6.10 branch, i also added a commit to change the branch/tags to reflect 6.10.0 only to realize that you had also done it. I rebased your change on top of that. So, essentially your change only now reflects the updating the dependencies. It would be appreciated if you could remove the "Update to 6.10.0" from the commit title.  Please feel free to merge when webengine is updated to 6.10.0.

hfiguiere commented:
there is no webengine....

See https://github.com/flathub/io.qt.qtwebengine.BaseApp/pull/393

hfiguiere commented:
See #29

Justinzobel commented:
On 3 November 2025 7:55:05 pm ACDT, Shyamnath Premnadh ***@***.***> wrote:
>@shyamnathp approved this pull request.
>
>Hey, thanks for the patch. When i created the 6.10 branch, i also added a commit to change the branch/tags to reflect 6.10.0 only to realize that you had also done it. I rebased your change on top of that. So, essentially your change only now reflects the updating the dependencies. It would be appreciated if you could remove the "Update to 6.10.0" from the commit title.  Please feel free to merge when webengine is updated to 6.10.0. 
>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/flathub/io.qt.PySide.BaseApp/pull/28#pullrequestreview-3410111788
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
I'll update this when I'm at my PC later today.

Justinzobel commented:
On 4 November 2025 12:15:57 am ACDT, "Hubert Figuière" ***@***.***> wrote:
>hfiguiere left a comment (flathub/io.qt.PySide.BaseApp#28)
>
>there is no webengine....
>
>See https://github.com/flathub/io.qt.qtwebengine.BaseApp/pull/393
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/flathub/io.qt.PySide.BaseApp/pull/28#issuecomment-3480606235
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
<https://github.com/flathub/io.qt.PySide.BaseApp/pull/28#issuecomment-3477907252>

Justinzobel commented:
I think I fixed it but maybe introduced a conflict. I wish I was better at git :laughing:

PunkPangolin commented:
bot, build

PunkPangolin commented:
QtWebEngine is available now, so maybe it works

PunkPangolin commented:
`llvm21` (or 20) has to be used; 18 is not available with freedektop sdk 25.08.

Commit:
Move to LLVM 21
